src/views/Article/Spinner.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import spinner from './spinner.gif';
+
+const Spinner = () => {
```

Comment:
Hello @Amoswachira ,
You had already defined a similar component in the components..Please reuse it to avoid code duplication
You should also treat views as only those components that connect to the store

Reply:
@Ogutu-Brian I will make that spinner reusable thanks